### PR TITLE
feat: wire-up thorchain lending My Loan Info

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

Does what it says on the box - wires up THORCHain lending "My Loan Info" section with actual data from THOR API:

 - Collateral Balance
 - Collateral Value
 - Debt Balance
 - Repayment lock in days

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

## Risk

None as this is under flag

## Testing

- With the lending flag on, ensure things are looking good in the "My Loan Info" section. Should zero out without a position.
- Open a loan from THORSwap and refresh the app, ensuring the loan data is displayed and looks sane

### Engineering

- ☝🏽 

### Operations

- ☝🏽 

## Screenshots (if applicable)

https://github.com/shapeshift/web/assets/17035424/5d8230d0-f441-422d-a8a3-93e396188109

image

### CODE REVIEW

**Comment by gomesalexandre** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5544** <a href="https://app.graphite.dev/github/pr/shapeshift/web/5544" target="_blank">Graphite</a> 
    * **PR #5545** <a href="https://app.graphite.dev/github/pr/shapeshift/web/5545" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/5545?utm_source=stack-comment).

File: src/pages/Lending/Pool/Pool.tsx [hunk 1]
```diff
@@ -55,24 +64,81 @@ const PoolHeader = () => {
 
 const flexDirPool: ResponsiveValue<Property.FlexDirection> = { base: 'column', lg: 'row' }
 
+// TODO(gomes): programmatic
```
Comment (gomesalexandre) [inline src/pages/Lending/Pool/Pool.tsx:67]: (not in this PR)

Comment (gomesalexandre) [reply src/pages/Lending/Pool/Pool.tsx]: (actually in this PR)

File: src/pages/Lending/Pool/Pool.tsx [hunk 2]
```diff
@@ -56,31 +69,112 @@ const PoolHeader = () => {
...
+    queryFn: async ({ queryKey }) => {
+      const [, { accountId, assetId }] = queryKey
+      const position = await getThorchainLendingPosition({ accountId, assetId })
+      return position
+    },
+    select: data => {
+      // returns actual derived data, or zero's out fields in case there is no active position
+      const collateralBalanceCryptoPrecision = fromThorBaseUnit(data?.collateral_current).toString()
+
+      const collateralBalanceFiatUserCurrency = fromThorBaseUnit(data?.collateral_current)
```
Comment (gomesalexandre) [inline src/pages/Lending/Pool/Pool.tsx:101]: derp, this should *always* be denominated in USD

```suggestion
      const collateralBalanceUsd = fromThorBaseUnit(data?.collateral_current)
```

**Review by 0xApotheosis — APPROVED** [review]
Tidy ser.

Opened a loan on Thorswap and it correctly populates in our UI 👌 

Screenshot 2023-10-27 at 11 58 31 am

**Pull request merged**